sdk_container/src/third_party/coreos-overlay/coreos/config/env/sys-boot/grub

```diff
@@ -12,6 +12,32 @@ cros_pre_src_prepare_adjust_version() {
 	sed -i "/AC_INIT/s/\b${PV//./\\.}\b/\0-${FLATCAR_VERSION}/g" configure.ac || die
 }
 
+# Prevent developer test modules from being built. These are normally always
+# installed, even by grub-install, but they have no use outside of testing and
+# take up valuable space in /boot. The best way to identify these is to look for
+# references to the tests/ directory.
+cros_post_src_prepare_drop_tests() {
+	gawk -i inplace '
```

Comment:
Does `-i` have the same meaning in `gawk` as it has in `sed`? Checking `gawk` manual tells me that `-i` is a shortcut for `--include`, so here you are doing an include of a file called `inplace`? Is there a chance that you actually are not modifying the Makefile? The image diff reports would tell you if some module files were gone from initrd, but I can't find those reports anywhere.

Reply:
The inplace feature is implemented through an additional gawk module that you need to load with `-i`. It's not built-in like it is with sed. I know it's working because the image diff shows this and more:

```
23:57:31  @@ -232,22 +221,15 @@
23:57:31   /boot/flatcar/grub/i386-pc/search_part_uuid.mod
23:57:31   /boot/flatcar/grub/i386-pc/sendkey.mod
23:57:31   /boot/flatcar/grub/i386-pc/setjmp.mod
23:57:31  -/boot/flatcar/grub/i386-pc/setjmp_test.mod
23:57:31   /boot/flatcar/grub/i386-pc/setpci.mod
23:57:31   /boot/flatcar/grub/i386-pc/sfs.mod
23:57:31  -/boot/flatcar/grub/i386-pc/shift_test.mod
23:57:31  -/boot/flatcar/grub/i386-pc/signature_test.mod
23:57:31   /boot/flatcar/grub/i386-pc/sleep.mod
23:57:31  -/boot/flatcar/grub/i386-pc/sleep_test.mod
```